### PR TITLE
New dnf changelog plugin

### PR DESCRIPTION
**Pull request by m-blaha**
For viewing the packages changelogs.

### CODE REVIEW

File: plugins/changelog.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+                           self.opts.count).format(self.opts.count))
+        else:
+            logger.info(_('Listing all changelogs'))
+
+        by_srpm = self.by_srpm(self.query())
+        for name in by_srpm:
+            print(_('Changelogs for {}').format(
+                ', '.join([str(pkg) for pkg in by_srpm[name]])))
+            for chlog in self.filter_changelogs(by_srpm[name][0]):
+                print(self.base.format_changelog(chlog))
```
Comment (j-mracek) [inline plugins/changelog.py]: Printing line by line could slow the operation.

File: dnf-plugins-core.spec [hunk 2]
```diff
@@ -56,6 +57,7 @@ Provides:       dnf-plugins-extras-repoclosure = %{version}-%{release}
 Provides:       dnf-plugins-extras-repograph = %{version}-%{release}
 Provides:       dnf-plugins-extras-repomanage = %{version}-%{release}
 Provides:       dnf-plugin-builddep = %{version}-%{release}
+Provides:       dnf-plugin-changelog = %{version}-%{release}
```
[the remark this answers is not in the document]
Comment (j-mracek) [reply dnf-plugins-core.spec]: I believe that most of provides was created historically. Like that in past it was a separate plugin, then it was incorporated into dnf-plugins-core. This is a political decision rather that technical one.

**Comment by j-mracek** [comment]
The code looks good. Only I would like to discuss the provide dnf-plugin-changelog = %{version}-%{release} if it is needed.

**Comment by m-blaha** [comment]
I have no problem with removing this provide.

**Pull request merged**